dgv/monitoring/DAG-hvd.py

```diff
@@ -0,0 +1,185 @@
+from datetime import timedelta, datetime
+import pandas as pd
+import requests
+from unidecode import unidecode
+from io import StringIO
+
+from airflow.models import DAG
+from airflow.operators.python import PythonOperator
+from airflow.operators.bash import BashOperator
+from airflow.utils.dates import days_ago
+from datagouvfr_data_pipelines.config import (
+    AIRFLOW_DAG_TMP,
+    MINIO_BUCKET_DATA_PIPELINE_OPEN,
+    MATTERMOST_MODERATION_NOUVEAUTES,
+)
+from datagouvfr_data_pipelines.utils.mattermost import send_message
+from datagouvfr_data_pipelines.utils.minio import MinIOClient
+
+DAG_NAME = "dgv_hvd"
+DATADIR = f"{AIRFLOW_DAG_TMP}{DAG_NAME}/data/"
+minio_open = MinIOClient(bucket=MINIO_BUCKET_DATA_PIPELINE_OPEN)
+
+
+def slugify(s):
+    return unidecode(s.lower().replace(" ", "-").replace("'", "-"))
+
+
+def get_hvd(ti):
+    print("Getting suivi ouverture")
+    ouverture_hvd_api = 'https://ouverture.data.gouv.fr/api/high_value_datasets'
+    df_ouverture = pd.DataFrame(requests.get(ouverture_hvd_api).json())
+    goal = len(df_ouverture)
```

Comment:
I think it's the number of *unique* hvd name? I mean we could compare the number of HVD "ensemble de données" for which we have at least one existing dataset.